source.go

```diff
@@ -157,42 +148,28 @@ func (s *Source) Teardown(ctx context.Context) error {
 }
 
 func (s *Source) LifecycleOnDeleted(ctx context.Context, cfg config.Config) error {
-	if err := s.Configure(ctx, cfg); err != nil {
-		return fmt.Errorf("fail to handle lifecycle delete event: %w", err)
-	}
-
-	// N.B. This should not stay in here for long, enrich the default.
-	//      Events are not passed enriched config with defaults.
-	params := s.config.Parameters()
-
-	if _, ok := cfg["logrepl.autoCleanup"]; !ok { // not set
-		s.config.LogreplAutoCleanup = params["logrepl.autoCleanup"].Default == "true"
-	}
-
-	if _, ok := cfg["logrepl.slotName"]; !ok {
-		s.config.LogreplSlotName = params["logrepl.slotName"].Default
-	}
-
-	if _, ok := cfg["logrepl.publicationName"]; !ok {
```

Comment:
These all get defaulted?

Reply:
`sdk.Util.ParseConfig` should take care of that, yes